ij/src/main/kotlin/org/jetbrains/academy/test/system/ij/analyzer/IjCodeAnalyzerUtil.kt

```diff
@@ -0,0 +1,81 @@
+package org.jetbrains.academy.test.system.ij.analyzer
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiFileFactory
+import com.intellij.psi.util.PsiTreeUtil
+import org.jetbrains.academy.test.system.ij.formatting.formatting
+import org.jetbrains.kotlin.idea.KotlinFileType
+import org.jetbrains.kotlin.psi.*
+
+/** Extracts [kotlin elements][KtElement] of given type from kotlin related files in project. */
+/** Extracts elements of given type from [PsiElement] subtree. */
+fun <T : PsiElement> PsiElement.extractElementsOfTypes(psiElementClassList: List<Class<out T>>): MutableCollection<T> =
```

Comment:
I'd prefer to change  `psiElementClassList: List<Class<out T>>` to `vararg`, it will help to avoid creating lists with one argument during calling this function, what do you think?